googletest/scripts/fuse_gtest_files.py

```diff
@@ -90,10 +93,10 @@ def VerifyFileExists(directory, relative_path):
   """
 
   if not os.path.isfile(os.path.join(directory, relative_path)):
-    print 'ERROR: Cannot find %s in directory %s.' % (relative_path,
-                                                      directory)
-    print ('Please either specify a valid project root directory '
-           'or omit it on the command line.')
+    print('ERROR: Cannot find %s in directory %s.' % (relative_path,
```

Comment:
Nope, that's only for *enforcing* new style syntax. Haven't done that as I've tried to keep python2.3 support (the print_function future is only available in 2.6+) ;)